cogs/moderation.py

```diff
@@ -492,6 +572,184 @@ async def on_member_kick(self, target, entry):
         )
         await self.save_action(action)
 
+    @commands.hybrid_group(
+        aliases=("emergency-staff", "alert", "alert-staff"),
+        cooldown_after_parsing=True,
+        fallback="send",
+        invoke_without_subcommand=True,
+    )
+    @commands.cooldown(1, EMERGENCY_COOLDOWN_HOURS * 60 * 60, commands.BucketType.guild)  # Cooldown per guild
+    @commands.guild_only()
+    async def emergency(self, ctx: GuiduckContext, *, reason: str):
```

Comment:
This is a really long function, I'd prefer if you can split it up

Reply:
Will do!

Reply:
Done, does this look good?